File: EducationApp.BusinessLogicLayer/Extention/BaseFilter/BaseFilterModel.cs
using static EducationApp.BusinessLogicLayer.Models.Enums.Enums;

namespace EducationApp.BusinessLogicLayer.Extention.BaseFilter
{

    public class BaseFilterModel
    {
        public string SearchString { get; set; }
        public int PageNumber { get; set; } = 1;
        public int PageSize { get; set; } = 10;
        public SortType SortType { get; set; }
    }
}

File: ClientApp/src/app/shared/models/Base/BaseFilterModel.ts
import { SortType } from 'src/app/shared/enums/SortType';
import { CurrencyType } from 'src/app/shared/enums/CurrencyType';

export class BaseFilterModel {
    
    searchString: string;
    pageNumber: number;
    pageSize: number;
    sortType: SortType;
    currencyType: CurrencyType;
   
}